frontend/src/components/filters/SingleSelect.tsx

```diff
@@ -1,84 +1,90 @@
-import { format } from 'date-fns';
+import { AvailabilityTypeName } from '@/common';
+
+// util
+import { getLabelFromValue } from '@/utils';
 
 export const SingleSelect = ({
-  label,
   field,
-  handleClose,
+  handleChange,
+  handleClose = () => {},
+  label,
+  openColor = 'default',
+  placeholder = 'placeholder',
+  useChip = false,
   value,
-  searchParams,
-  setSearchParams,
 }: {
-  label: string;
   field: any;
-  handleClose: () => void;
+  handleChange?: any;
+  handleClose?: () => void;
+  label?: string;
+  openColor?: string;
+  placeholder?: string;
+  useChip?: boolean;
   value?: string;
-  searchParams: URLSearchParams;
-  setSearchParams: (searchParams: any) => any;
 }) => {
-  const placeholder = 'Select availability type';
-  const onChange = (value: string) => {
-    const date = new Date();
-    searchParams.set(field.name, value);
-    if (!searchParams.get(Filters.AVAILABILITY_FROM_DATE)) {
-      searchParams.set(
-        Filters.AVAILABILITY_FROM_DATE,
-        format(
-          new Date(date.getFullYear(), date.getMonth(), date.getDate()),
-          'yyyy-MM-dd',
-        ),
-      );
-    }
-    if (!searchParams.get(Filters.AVAILABILITY_TO_DATE)) {
-      searchParams.set(
-        Filters.AVAILABILITY_TO_DATE,
-        format(
-          new Date(date.getFullYear(), date.getMonth(), date.getDate()),
-          'yyyy-MM-dd',
-        ),
-      );
-    }
-    setSearchParams({ ...Object.fromEntries(searchParams) });
-  };
+  // set menu style
+  let menuStyle = field.name ? menuItemClass[field.name] : classes.menu.container;
+  let menuOpenStyle = menuStyle;
+  let menuClosedStyle = menuStyle;
+  if (openColor === 'skyline') {
+    menuOpenStyle = menuStyle + ' border-skyline-200 border-2';
+  }
+
   return (
     <>
-      <span className="label">{label}</span>
+      {label && <span className="label">{label}</span>}
       <Menu as="div" className="relative inline-block text-left w-full">
         {({ open }) => (
           <>
             {value ? (
-              <div className={menuItemClass[field.name]}>
-                <Chip
-                  value={value}
-                  label={AvailabilityTypeName[value as AvailabilityType]}
-                  name={field.name}
-                  handleClose={handleClose}
-                />
-                <MenuButton aria-label="Single Select Menu Button" id={field.name}>
+              useChip ? (
+                <div className={menuStyle}>
+                  {' '}
+                  <Chip
+                    value={value}
+                    label={AvailabilityTypeName[value as AvailabilityType]}
+                    name={field.name}
+                    handleClose={handleClose}
+                  />
+                </div>
+              ) : (
+                <MenuButton
+                  aria-label="Single Select Menu Button"
+                  id={field.name}
+                  className={open ? menuOpenStyle : menuClosedStyle}
```

Comment:
{open ? ` ${menuStyle} ${styleProp} ` : menuStyle}

Reply:
follow up from our chat: 
You can either remove the blue outline since this seems inconsistent, or, pass in an optional style prop, and just set the class with something like: {open ? passed prop : menuStyle}